### DISCUSSION TITLE
Is it fine to use a modified version of the client when playing online?

### DISCUSSION THREAD

**Discussion by Delta-official** [ppy/osu#29698] [Q&A] [answered]
If one were to for example fork osu!lazer with the intent to disable a feature (e.g. kiai flashing), apply the needed changes, compile it and use it during online play, would that still be okay according to the rules or would it be considered illegal?

**Comment by AntiApple4life** [comment, marked as answer]
There are already checks in place to make sure the only versions of lazer that can play online are the binaries distributed in the releases section. If you try to compile the latest release yourself, it cannot play online. I would assume that if you compiled a modified build, it would just refuse to play online.

**Reply by Delta-official** [reply]
Hm. That sucks (seeing as I can't even change the cursor in Argon due to this), but I get the reason why it was done. Thanks for the answer.
